ecr/create-resource/main.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_ecr_repository" "service" {
+  name                 = "${var.repository_name}"
+  image_tag_mutability = "MUTABLE"
```

Comment:
I don't know if/when we would want this to be `IMMUTABLE`. Maybe it would be worth making it a variable?

Reply:
Do we want to be able to overwrite existing ECR images?  If not, the value should probably be IMMUTABLE.

Reply:
Probably not. Just switched it to IMMUTABLE.